desktop/core/src/desktop/js/apps/storageBrowser/SummaryModal/SummaryModal.tsx

```diff
@@ -0,0 +1,117 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import React, { useEffect, useState } from 'react';
+import Modal from 'cuix/dist/components/Modal';
+import { Row, Col, Spin } from 'antd';
+
+import huePubSub from '../../../utils/huePubSub';
+import { i18nReact } from '../../../utils/i18nReact';
+import { content_summary } from '../../../reactComponents/FileChooser/api';
+import './SummaryModal.scss';
+
+interface SummaryModalProps {
+  path: string;
+  showModal: boolean;
+  onClose: () => void;
+}
+
+const SummaryModal: React.FC<SummaryModalProps> = ({ showModal, onClose, path }): JSX.Element => {
+  const { t } = i18nReact.useTranslation();
+  const [loadingSummary, setLoadingSummary] = useState(true);
+  const [summary, setSummary] = useState([]);
+
+  const formatBytes = (bytes, decimals) => {
+    if (bytes == -1) {
+      return 'Not available';
+    }
+    if (bytes == 0) {
+      return '0 Byte';
+    }
+    const k = 1024;
+    const dm = decimals + 1 || 3;
```

Comment:
define decimal as `decimalPoints?: number`

and check if undefined to decide whether to use the default value of 2 decimalpoints.